index.js

```diff
@@ -1 +1,121 @@
 // implement your API here
+
+//this is how you import in express
+const express = require("express");
+const db = require("./data/db");
+
+const server = express();
+server.use(express.json());
+
+//http is protocol that connects client and server
+//send is a node function while get is http
+server.get("/", (req, res) => {
+  res.send("Hello World");
+});
+
+//GET 
+
+server.get("/api/users", (req, res) => {
+  db.find()
+    .then(users => {
+      if (users) {
+        res.status(200).json(users);
+      } else {
+        res
+          .status(404)
+          .json({ message: "The users information could not be retrieved." });
+      }
+    })
+    .catch(err => {
+      res
+        .status(500)
+        .json({ error: "The user information could not be retrieved." });
+    });
+});
+
+//GET ID
+
+server.get("/api/users/:id", (req, res) => {
+  const id = req.param.id;
```

Comment:
Close! Just one letter off

```suggestion
  const id = req.params.id;
```